docs/src/main/sphinx/sql/show-catalogs.rst

```diff
@@ -12,5 +12,11 @@ Synopsis
 Description
 -----------
 
-List the available catalogs.
-The ``LIKE`` clause can be used to restrict the list of catalog names.
+List the available catalogs. 
+
+:ref:`Specifying a pattern <like_operator>` allows you to filter the results to
```

Comment:
Let's use consistent terminology. It's not a "pattern" it's the `LIKE clause` (which accepts a pattern).

nit: like_operator should be called `like_clause` IMO.

Reply:
Thanks for the correction.

Regarding consistency in labeling, I see LIKE being referred to as both an operator and clause on the internet. Within our docs, though, the word operator is used to reference to LIKE, BETWEEN, IS NULL, and others, already.

Reply:
We could use

Specifying a pattern in the optional LIKE clause  .. that would be correct and consistent with the code snippet where `pattern` is used..